### ISSUE TITLE
Some questions for paper?

### ISSUE THREAD

**Issue by aptsunny** [AlgoHunt/Face-Xray#1]
Are there any considerations for hyperparameters configuration in this paper? (Lambda=100), How long does the amount of dataset and corresponding training time until convergence? Looking forward to your answer.

**Comment by niyunsheng** [comment]
I have implemented all the parts and experiments, the strange thing is that the loss of this xray is infinity, I do n’t know what the problem is

**Comment by ZuoyanL** [comment]
> I have implemented all the parts and experiments, the strange thing is that the loss of this xray is infinity, I do n’t know what the problem is

Hi friends, I have some problems with reproducing the face X-ray. Could you please share your experiment codes?

**Comment by niyunsheng** [comment]
我这个代码好久以前的了，我没有复现出来，我可能还得整理一下

**Comment by ZuoyanL** [comment]
> 我这个代码好久以前的了，我没有复现出来，我可能还得整理一下
> […](#)
> On Dec 27, 2020, at 8:58 PM, ZuoyanL ***@***.***> wrote: ﻿ I have implemented all the parts and experiments, the strange thing is that the loss of this xray is infinity, I do n’t know what the problem is Hi friends, I have some problems with reproducing the face X-ray. Could you please share your experiment codes? — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.
好的，  那你知道论文里面 DF NT F2F NT 的Face X-ray 是怎么构建的吗？似乎需要和Origin数据集做差得到？
